scripts/syntactics/index.rst

```diff
@@ -0,0 +1,77 @@
+Dependency Parsing
+---------------------------------
+
```

Comment:
Add download link by adding:
```
:download:`[Download] </model_zoo/syntactics.zip>`
```

Reply:
Now link has been added, but NMT link broke, failed the test.

```
Warning, treated as error:

/var/lib/jenkins/workspace/gluon-nlp-docs/docs/model_zoo/machine_translation/index.rst.rst:43:broken link: https://apache-mxnet.s3.cn-north-1.amazonaws.com.cn/gluon/models/transformer_en_de_512_WMT2014-14bd361b.zip (('Connection broken: OSError("(104, \'ECONNRESET\')",)', OSError("(104, 'ECONNRESET')",)))

Makefile:171: recipe for target 'linkcheck' failed
```

It must be occasionally, I can download it using browser. What shall I do?

Reply:
@hankcs thanks. Sorry about it as we just recently added link check and are still ironing out these intermittent issues. For now, you can delete the last line of https://github.com/dmlc/gluon-nlp/blob/master/scripts/machine_translation/index.rst. I will address it in a separate PR.

Reply:
Or, you can wait for https://github.com/dmlc/gluon-nlp/pull/414 in which the linkcheck is not enforced on PRs.

Reply:
> @hankcs thanks. Sorry about it as we just recently added link check and are still ironing out these intermittent issues. For now, you can delete the last line of https://github.com/dmlc/gluon-nlp/blob/master/scripts/machine_translation/index.rst. I will address it in a separate PR.

I've temporarily delete that link, thanks.